recipe/meta.yaml

```diff
@@ -13,12 +13,13 @@ source:
     - vc14.patch
 
 build:
-  number: 3
+  number: 4
   binary_relocation: true
   features:
     - vc9  # [win and py27]
     - vc10  # [win and py34]
-    - vc14  # [win and py35]
+    - vc14  # [win and py>=35]
+  skip: true  # [win and py36]
```

Comment:
Honestly it might be better to skip `py35` as we will likely be dropping Python 3.5 soon given Python 3.7 is right around the corner.